src/02-common-principles.md

```diff
@@ -644,6 +540,147 @@ for more information.
 }
 ```
 
+## The Inheritance Principle
+
+1.  Any metadata file (such as `.json`, `.bvec` or `.tsv`) MAY be defined at any directory level.
+
+1.  For a given data file, any metadata file is applicable to that data file if:
```

Comment:
although markdown does numbering for us, since there are direct references to them in the text (e.g. `(rule 5.2.2)`) I think it would be better just to number them here as well, so it is easier to refer back while reading markdown and not rendered version

Reply:
Better would be enumeration cross-references; but as far as I can tell no standardized solution for such exists?

Reply:
> it would be better just to number them here as well

Not permitted: Lestropie/bids-specification#2

Reply:
eh, thanks for trying it out! 
Indeed from https://www.npmjs.com/package/remark-lint-ordered-list-marker-value I see that
- the default is `"one"`
- there seems to be no I do not see some way to override locally in a file/for this specific list.
   - https://stackoverflow.com/a/63753061/1265472 suggests that a local to a directory `.remarkrc` could be used to overload some settings, but that would not help here/more confusing than helpful in the long run
- the other places in BIDS specification follow the "one" convention and there it seems to make sense

so I guess I will "retract" my invaluable comment ;-) Thanks again for trying

Reply:
actually according to https://github.com/remarkjs/remark-lint#ignore-warnings there is an inline way to disable/re-enable specific warnings in a specific block, but I do not see there a way to change configuration setting (i.e. make it "ordered") . I will suggest this disable/reenable in the PR just to see if works -- not sure if worth it if we can't lint the order